extensions/amp-form/0.1/amp-form.js

```diff
@@ -443,7 +377,7 @@ function getUserValidityStateFor(element) {
 function checkUserValidity(element, propagate = false) {
```

Comment:
Possibly yes. I'll file an issue to refactor that into FV in a follow up PR. For now `FormValidators` are actually `FormValidationReporters` mostly.